### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tarekait1996**
* update README.md with the new features
* also added TRAVISCI as part of the technologies used for the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +51,8 @@ A social platform for ranting about society. Users will see the most triggering
 * [Express - framework for NodeJS to sweet things up](https://expressjs.com/en/guide/routing.html)
 * [Angular - for the UI ](https://material.angular.io/)
 * [Node.js](https://nodejs.org/api/)
-
+###Testing
```
Comment (paulMoose) [inline README.md]: Good job! Could you please add a Space after the three # so that it shows as a header.

**Comment by paulMoose** [comment]
Nice job! I added a comment on one of the lines. It is showing like this in the preview
image

File: README.md (hunk 1 above)
Comment (tarekait1996) [reply README.md]: done!

Comment (paulMoose) [reply README.md]: Perfect, you can merge both your PRs.

**Pull request merged**